CMakeLists.txt

```diff
@@ -205,6 +207,98 @@ if (NN_ENABLE_NANOCAT)
     target_link_libraries (nanocat ${PROJECT_NAME})
 endif ()
 
+find_program (ASCIIDOCTOR_EXE asciidoctor)
```

Comment:
The above pseudo code is wrong, since we want docs enabled by default, but don't want default options to fatal error.  So I've found and delivered a different solution to this already. :-)